airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json

```diff
@@ -211,6 +206,18 @@
         "default": 15,
         "examples": ["15"],
         "order": 6
+      },
+      "use_1s1t_format": {
+        "type": "boolean",
+        "description": "(Beta) Use <a href=\"https://github.com/airbytehq/airbyte/issues/26028\" target=\"_blank\">Destinations V2</a>. Contact Airbyte Support to participate in the beta program.",
+        "title": "Use Destinations V2 (Early Access)",
+        "order": 7
+      },
+      "raw_data_dataset": {
+        "type": "string",
+        "description": "(Beta) The dataset to write raw tables into",
```

Comment:
Doesn't have to be in this change, but should we eventually have a longer description about why a user might want to put something here?

Reply:
yeah, I figure we can workshop the wording closer to full release. there's probably a lot of stuff that we should polish here, e.g. adding default values and stuff :shrug: